bundles/framework/hierarchical-layerlist/service/LayerlistExtenderService.js

```diff
@@ -0,0 +1,424 @@
+/**
+ * @class Oskari.framework.bundle.hierarchical-layerlist.OskariEventNotifierServic
```

Comment:
Update this documentation. It says 'Oskari.framework.bundle.hierarchical-layerlist.LayerlistExtenderService' below.